pkg/cloudprovider/launchtemplate.go

```diff
@@ -145,6 +145,7 @@ func (p *LaunchTemplateProvider) Get(ctx context.Context, provider *v1alpha1.AWS
 func (p *LaunchTemplateProvider) ensureLaunchTemplate(ctx context.Context, options *amifamily.LaunchTemplate) (*ec2.LaunchTemplate, error) {
 	var launchTemplate *ec2.LaunchTemplate
 	name := launchTemplateName(options)
+	ctx = logging.WithLogger(ctx, logging.FromContext(ctx).With("template-name", name))
```

Comment:
```suggestion
	ctx = logging.WithLogger(ctx, logging.FromContext(ctx).With("launch-template-name", name))
```

We should be consistent with the naming of these structured fields